apps/auth-server/controllers/auth/required.js

```diff
@@ -11,12 +11,16 @@ exports.index = (req, res, next) => {
 
   const config = req.client.config ? req.client.config : {};
   const configRequiredFields = config && config.requiredFields ? config.requiredFields : {};
-  
+
+  const requiredUserFieldsLabels = config && config.requiredFields && config.requiredFields.requiredUserFieldsLabels ? config.requiredFields.requiredUserFieldsLabels : {};
+
   // Replace field labels with labels defined in the client config (if provided)
-  if (configRequiredFields && configRequiredFields.labels) {
+  if (requiredUserFieldsLabels && Object.keys(requiredUserFieldsLabels).length > 0) {
     requiredUserFields = requiredUserFields.map((field) => {
-      if (configRequiredFields.labels[field.key]) {
-        field.label = configRequiredFields.labels[field.key];
+      const newLabel = requiredUserFieldsLabels[field.id] ?? '';
+
+      if (!!newLabel) {
+        field.label = newLabel;
```

Comment:
Als het label per default een lege string is, dan hoef je niet perse een if te gebruiken.

Reply:
Dit is voor het overschrijven van de labels. Dus field.label kan bijvoorbeeld 'Naam' zijn die wil je alleen overschrijven als er een nieuw label is :)